### PR TITLE
Feature/rubocop upgrade

### PR DESCRIPTION
**Pull request by qbantek**

### CODE REVIEW

File: spec/rest/verix/verix_documents_spec.rb [hunk 1]
```diff
@@ -22,18 +23,19 @@
         :record_id
       )
     end
+
     it_behaves_like 'an API request'
     it_behaves_like 'a successful request'
   end
 
   describe '#show' do
```
Comment (sebeth-fa) [inline spec/rest/verix/verix_documents_spec.rb:31]: This no longer tests both use cases where record_id is nil or where document_id is nil. Is this intentional?

File: spec/rest/verix/verix_pdf_documents_spec.rb [hunk 2]
```diff
@@ -7,16 +7,16 @@
   include SpecHelpers::Client
 
   let(:api_client) { client }
-  let(:document) { FinApps::REST::VerixPdfDocuments.new(api_client) }
+  let(:document) { described_class.new(api_client) }
 
   describe '#show' do
     context 'when missing parameters' do
-      subject { document.show(nil, :provider_id) }
```
Comment (sebeth-fa) [inline spec/rest/verix/verix_pdf_documents_spec.rb:14]: removes record_id = nil use case

File: spec/rest/verix/verix_documents_spec.rb [hunk 3]
```diff
@@ -22,18 +23,19 @@
...
     context 'when missing parameters' do
-      subject { document.show(nil, :document_id) }
+      subject { document.show(:record_id, nil) }
+
       it 'raises an error when missing record id' do
         expect { subject }.to raise_error(FinAppsCore::MissingArgumentsError)
       end
 
-      subject { document.show(:record_id, nil) }
```
Comment (sebeth-fa) [inline spec/rest/verix/verix_documents_spec.rb:36]: Removes document_id = nil use case

File: spec/rest/verix/verix_pdf_documents_spec.rb (hunk 2 above)
Comment (qbantek) [reply spec/rest/verix/verix_pdf_documents_spec.rb:14]: Fixed

Comment (qbantek) [reply spec/rest/verix/verix_pdf_documents_spec.rb:14]: Fixed

File: spec/rest/verix/verix_documents_spec.rb (hunk 3 above)
Comment (qbantek) [reply spec/rest/verix/verix_documents_spec.rb:36]: Fixed

**Pull request merged**